INSTALLATION.md

```diff
@@ -1,259 +1,250 @@
-# Talawa-API Installation
```

Comment:
We need the Talawa API Installation title to remain as it describes the purpose of  the document.
We have two installation methods. Standard and Docker. These need to have section headings like the current INSTALLATION.md file has.

Please also add  a table of contents like we have in the CONTRIBUTING.md file. It will make navigation easier.